README.md

```diff
@@ -14,31 +14,11 @@ Application to visualize animated 3D medical data using Hololens, with shared ex
 
   * Convert your input models into Asset Bundles.
 
-      * You can import models as a series of VTK models. The [input format specification is here](https://github.com/MicroscopeIT/holo/tree/master/Input%20documentation). Example VTK model is inside `Test Model` subdirectory there. This is the advised format, that supports animation using blend shapes.
-
-      * Or you can import models with each layer loaded from Unity assets (so it can be .obj, .prefab or any other model format supported by Unity). Example is inside `unity/Holo/Assets/Test Models/Skull/` in this repository.
+		The whole pipeline for converting your models into uploadable Asset Bundles can be found [here](https://github.com/MicroscopeIT/holo-preprocess/)
```

Comment:
Please change link to: https://github.com/tooploox/holo-preprocess instead of MicroscopeIT